ehitchhiking-ui/src/app/shared/components/trips/trips.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import { LoaderSize } from "../../enums/pre-loader-sizes";
+import { MatDialogRef } from "@angular/material";
+import { Trip } from "./trips";
+
+@Component({
+  selector: 'app-trips',
+  templateUrl: './trips.component.html',
+  styleUrls: ['./trips.component.sass']
+})
+export class TripsComponent implements OnInit {
+  StatusEnum = Object.freeze({"active":0, "completed":1, "declined":2});
+  trips: Trip[] = [
+    {id: 0, from_: 'Mstislavca1', to: 'Turovsky', isFavorite: false, status: this.StatusEnum.active },
+    {id: 1,from_: 'Mstislavca2', to: 'Turovsky', isFavorite: false, status: this.StatusEnum.active},
+    {id: 2, from_: 'Mstislavca3', to: 'Turovsky', isFavorite: false, status: this.StatusEnum.completed},
+    {id: 3, from_: 'Mstislavca4', to: 'Turovsky', isFavorite: false, status: this.StatusEnum.completed},
+    {id: 4, from_: 'Mstislavca5', to: 'Turovsky', isFavorite: false, status: this.StatusEnum.declined},
+    {id: 5, from_: 'Mstislavca5', to: 'Turovsky', isFavorite: false, status: this.StatusEnum.completed},
+    {id: 6, from_: 'Mstislavca6', to: 'Turovsky', isFavorite: false, status: this.StatusEnum.declined},
+  ];
+  begin = 0;
+  end = 5;
+  tripsArray = [];
+  size: LoaderSize = LoaderSize.Large;
+  loading = true;
```

Comment:
not used

Reply:
Pls add pre-loader to your component. Feel free to ask @AnastasiaHarost how to do that

Reply:
added pre-loader